SPECS/prometheus-node-exporter/prometheus-node-exporter.spec

```diff
@@ -73,9 +76,6 @@ install -Dpm0644 example-rules.yml %{buildroot}%{_datadir}/prometheus/node-expor
 install -Dpm0644 %{SOURCE5} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
 mkdir -vp %{buildroot}%{_sharedstatedir}/prometheus/node-exporter
 
-%check
-make test
```

Comment:
Why are we removing the tests?

Reply:
Nevermind, I see you just moved them elsewhere. Not sure what the `bin/node_exporter --help` is for, though. And could we move the whole section where it was - we usually keep them after the `%install` section.

Reply:
The point is that "make test" did not catch the bug that this PR fixes, namely that the prometheus-node-exporter "binary" is not an executable file. Therefore I added a simple test that the binary can be run.

I can move the check section back if that's the house style.